wp-content/plugins/wporg-learn/inc/blocks.php

```diff
@@ -71,6 +186,10 @@ function register_workshop_details() {
  * @return string HTML output used by the block
  */
 function workshop_details_render_callback( $attributes, $content ) {
+	if ( get_post_type() !== 'wporg_workshop' ) {
```

Comment:
The lack of validation here recently lead to an incident where the Widgets editor crashed due to this block mistakenly being added to the Lesson Plan sidebar.